wasm/js/transform_avro/src/main.js

```diff
@@ -1,4 +1,4 @@
-const avro = require("avro-js");
+const avro = require("avsc");
```

Comment:
The only important change to this file is switching `avro-js` for `avsc` (see top post for link to benchmarks showing why). The rest of the changes are just name changes and could be left out. ended up including them because this branch was what I used as the basis for multiple attempts at getting both Avro and Protobuf schemas to work together in a single transform. This included reading from file, reading from registry, and various ways to include asynchronous calls in this file so that it would work after being successfully deployed.

[Here](https://github.com/vuldin/redpanda-examples/blob/two-transforms-fail/wasm/js/transform_avro/src/main.js) is one of those attempts where I'm using only synchronous code to successfully encode JSON to both Avro and Protobuf. The problem with this attempt is later where I'm adding two entries to the Map object that gets returned from `processRecord`. More details on the issue [here](https://github.com/redpanda-data/redpanda/issues/4090).

The renaming in this file would have made (and will make) way more sense with Avro and Protobuf transforms.

Reply:
Agree with the renaming +1

As for the issue you observed, I'm pretty sure I fixed that bug and the change landed in the `wasm-si-beta` release. When you initially ran into that issue did you use the old version that was listed in the `package.json` file in this project?

Reply:
> Agree with the renaming +1
> 
> As for the issue you observed, I'm pretty sure I fixed that bug and the change landed in the `wasm-si-beta` release. When you initially ran into that issue did you use the old version that was listed in the `package.json` file in this project?

Yep that was the issue! I posted more details [here](https://github.com/redpanda-data/redpanda/issues/4090).

Thanks for the review, I'll update the PR later today.